semgrep/semgrep/test.py

```diff
@@ -109,6 +116,8 @@ def line_has_ok(line: str) -> bool:
         or "# ok:" in line
         or "//ok:" in line
         or "// ok:" in line
+        or "<!--ok:" in line
+        or "<!-- ok:" in line
```

Comment:
Also allows `ok` in HTML comments

Reply:
Should we do the same for `line_has_todo_rule` and `line_has_todo_ok`?

Reply:
I'm not really sure we use the feature anymore, but I guess there's no harm in adding it 👍

Reply:
I see a few places:

```
$ rg 'todook' semgrep-rules | wc -l
3
$ rg 'todoruleid' semgrep-rules | wc -l
19
```